src/app/pages/AccountPage/Features/TransactionHistory/index.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-restricted-syntax */
```

Comment:
danger

Reply:
It was complaining for no good reason, so that's why I added it. Added "block" disable now, as `eslint-disable-next-line` was not picking it up.

Reply:
Ah, sorry it wasted your time. I'll fix linter later